live/src/js/helper/getMaxArrayView.js

```diff
@@ -2,6 +2,9 @@ const getMaxArrayView = (data) => {
 	let seperator = 0;
 	let limit = 16;
 	for (let i = 0; i < data.length && limit > 0; i += 1) {
+		if(null == data[i]) {
```

Comment:
A better way here would be to check for a truthy value of `data[i]` like:
```js
if(data[i]) {
	// rest of the code
}
```
This way we won't also need the `continue` :slightly_smiling_face: